R/content.R

```diff
@@ -1548,9 +1548,12 @@ search_content <- function(
     limit = limit
   )
 
-  purrr::map(res, function(x) {
+  content_list <- purrr::map(res, function(x) {
     Content$new(client, x)
   })
+
+  class(content_list) <- c("connect_content_list", class(content_list))
+  content_list
```

Comment:
Do we need to preface this class with `connect_`? That seems like unnecessary typing for us, and if we want to communicate this is from Connect, we can do that in the print method(s) on this object. Our R6 classes are _not_ for example all prefaced with `Connect_` 

I hate `content`, as always, but I don't think that's solvable here, but so much do I wish we had a better label for this.

Reply:
The S3 classes we've been playing with so far (just integrations and @karawoo's work on users right now) all use a `connect_` prefix.

My thinking is that S3 classes are more common than R6 classes and are a lot less strict in their form, so having more verbosely & explicitly-named classes here is better.

There wasn't much _discussion_ of this approach at the time, so I didn't think it was controversial! (integrations: #431, #437, #440, user: #462)

If we conclude that it should just be named `content_list`, I feel like I should also change the name of the integrations classes.